node/src/actors/peers_manager/handlers.rs

```diff
@@ -18,6 +18,22 @@ impl Handler<AddPeers> for PeersManager {
     }
 }
 
+/// Handler for AddPeers message
+impl Handler<MeltPeers> for PeersManager {
+    type Result = Result<(), failure::Error>;
+
+    fn handle(&mut self, _msg: MeltPeers, _: &mut Context<Self>) -> Self::Result {
+        // Melt addresses
+        let addresses = self.peers.extract_melted_peers_from_ice_bucket();
```

Comment:
The method name is more descriptive than the comment :laughing: 

```suggestion
        let addresses = self.peers.extract_melted_peers_from_ice_bucket();
```